psalm.xml

```diff
@@ -6,6 +6,8 @@
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
     errorBaseline="baseline.xml"
+    findUnusedBaselineEntry="true"
+    findUnusedCode="false"
```

Comment:
These would probably be useful – if they work …

Reply:
I ran into the following issue: https://github.com/psalm/psalm-plugin-symfony/issues/151

Reply:
I ran into this issue: https://github.com/psalm/psalm-plugin-symfony/issues/151